web/settings.py

```diff
@@ -74,6 +74,7 @@
     "django.contrib.auth.middleware.AuthenticationMiddleware",
     "django.contrib.messages.middleware.MessageMiddleware",
     "django.middleware.clickjacking.XFrameOptionsMiddleware",
+    "app.securemiddleware.set_secure_headers",
```

Comment:
Pretty sure this path isn't correct. There isn't even an `app` module extant. Ours is called `web`. I think you did not run the code to check if it works.